mmdeploy/codebase/mmocr/deploy/mmocr.py

```diff
@@ -18,6 +18,15 @@ class MMOCR(MMCodebase):
 
     task_registry = MMOCR_TASK
 
+    @classmethod
+    def import_module(cls):
```

Comment:
As this function is used to invoke `register_all_modules`, I think we can rename it to `register_all_modules` so user won't get confused.